ska_conda/ska_builder.py

```diff
@@ -10,43 +10,28 @@
 
 class SkaBuilder(object):
 
-    def __init__(self, build_root='.', user='sot', git_repo_path='git@github.com:{user}/{name}.git'):
-        self.user = user
-        self.git_repo_path = git_repo_path
+    def __init__(self, build_root='.'):
         self.build_dir = os.path.abspath(os.path.join(build_root, "builds"))
         self.src_dir = os.path.abspath(os.path.join(build_root, "src"))
         os.environ["SKA_TOP_SRC_DIR"] = self.src_dir
 
     def _clone_repo(self, name, tag=None):
         if name in no_source_pkgs:
             return
-        print("Cloning source %s." % name)
+        print("Cloning or updating source source %s." % name)
         clone_path = os.path.join(self.src_dir, name)
         if not os.path.exists(clone_path):
-            # Try ssh first to avoid needing passwords for the private repos
-            # We could add these ssh strings to the meta.yaml for convenience
-            # The default ssh is not going to work if the package name doesn't match the
-            # top-level git repo name (cmd_states, eng_archive)
-            try:
-                repo = git.Repo.clone_from(self.git_repo_path.format(user=self.user, name=name), clone_path)
-                assert not repo.bare
-                print("Cloned via default ssh git")
-            except:
-                yml = os.path.join(pkg_defs_path, name, "meta.yaml")
-                with open(yml) as f:
-                    requires = False
-                    while not requires:
-                        line = f.readline().strip()
-                        if line.startswith("path:"):
-                            requires = True
-                    data = yaml.load(f)
-                    url = data['about']['home']
-                repo = git.Repo.clone_from(url, clone_path)
-                print("Cloned from url {}".format(url))
+            metayml = os.path.join(pkg_defs_path, name, "meta.yaml")
+            # It isn't clean yaml at this point, so just extract the string we want after "home:"
```

Comment:
It looks like the previous `requires/path` loop was dealing with the fact that the meta.yaml isn't compliant YAML at this point at the `path` key.  I just went with a regex here to grab the string after `home:` instead of trying to use the YAML parser.  I think that is fair for our application.

Reply:
I suspect it could be made into compliant YAML by enclosing the `{{ .. }}/..` stuff in double quotes, but I agree with your point to just get the job done here.